src/puppetlabs/puppetdb/query_eng/engine.clj

```diff
@@ -667,13 +667,31 @@
         [:null name]
         [field name]))))
 
+(defn merge-function-options
+  "Optionally merge call and grouping into an existing query map.
+   Alias function calls with escape-quoted function name for hsqldb compatability."
+  [m call grouping]
```

Comment:
Could we have a schema in here or something in the docstring to show what a call might look like?